test/cpp/microbenchmarks/bm_chttp2_transport.cc

```diff
@@ -565,6 +565,7 @@ static void BM_TransportStreamRecv(benchmark::State& state) {
   grpc_transport_stream_op_batch_payload op_payload(nullptr);
   grpc_transport_stream_op_batch op;
   grpc_core::OrphanablePtr<grpc_core::ByteStream> recv_stream;
+  bool recv_error = false;
```

Comment:
If the transports check for this being null, then there won't be any need to set this here.